valusets/ValueSet-DM-medicationrequest-course-of-therapy.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="c6c997f7-6b5c-4d92-85a5-9199cd2b17e8" />
+    <url value="https://fhir.nhs.uk/ValueSet/DM-medicationrequest-course-of-therapy" />
+    <version value="1.0.0" />
+    <name value="MedicationRequestCourseOfTherapyCodes" />
+    <title value="Medication Request  Course of  Therapy  Codes" />
+    <status value="draft" />
+    <experimental value="false" />
+    <date value="2020-07-02T09:29:23+11:00" />
+    <publisher value="FHIR Project team" />
+    <contact>
+        <telecom>
```

Comment:
Telecom details needs to be fully updated

Reply:
Added telecom details